Charts: update Stock demo description (#1985)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/Stock/description.md

A stock [series type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. You can specify the stock series type for each object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[stock](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/StockSeries/) object to configure properties of the Chart stock series.

> In stock series, the bottom and top values of a vertical line correspond to high and low prices. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for high and low prices, respectively.

Reviewer: ```suggestion
In stock series, the bottom and top values of a vertical line correspond to high and low prices. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for high and low prices.
```

> In stock series, high and low prices are represented by the bottom and top values of the vertical line. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for high and low prices respectively.

Reviewer 2: **Represents**. No exceptions: http://tw.devexpress.devx/language-guidelines/misused-words/represent/

> respectively

"Respectively" is a clause that should be separated by a comma.

```suggestion
In stock series, the bottom and top values of a vertical line correspond to high and low prices. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for high and low prices, respectively.
```

> Open and close prices are represented by left and right tick marks. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties.

Reviewer 2: Likewise, we cannot have this word in our documentation: http://tw.devexpress.devx/language-guidelines/misused-words/represent/

```suggestion
Open and close prices are displayed as left and right tick marks. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties.
```

> To mirror the changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To specify whether high, low, open or close prices of points the component should compare, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.

Reviewer 2: The words like "whether" or "if" have binary logic, basically, you can answer such a question only with "yes" or "no". However, you list four items after "whether", which is definitely incorrect.

For example, we can rewrite this paragraph as follows:

```suggestion
To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To switch between high, low, open or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
```
